guides/common/modules/proc_enabling-rh-cloud-and-insights-client-reports-on-hosts.adoc

```diff
@@ -0,0 +1,20 @@
+[id="enabling-rh-cloud-and-insights-client-reports-on-hosts_{context}"]
+= Enabling RH Cloud and Insights client reports on hosts
+
+You can enable the Red Hat Insights client on hosts and have the hosts send RH Cloud reports to the {RHCloud}.
+
+ifdef::katello,foreman-el,foreman-deb[]
+Red Hat Insights is a service by Red Hat for {RHEL} hosts.
+Ensure to set this parameter for {RHEL} hosts only.
+If you set the parameter on any non-{RHEL} operating systems, it automatically uploads new reports to the {RHCloud}.
+endif::[]
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Host* > *Provisioning Setup* > *Operating Systems*.
+. Select any {RHEL} operating systems for which you want to change the value.
+. On the *Parameters* tab, click on the edit button of `host_registration_insights`.
```

Comment:
```suggestion
. On the *Parameters* tab, click on the edit button of the `host_registration_insights` parameter.
```

Reply:
Btw, there's no an edit button for parameters:
![image](https://github.com/user-attachments/assets/28ffa766-009d-450f-9069-1b337340dfb5)

If users are enabling it for the OS, they will probably have to add the parameter, not edit it.

Reply:
Yes, verified and updated. I have reworded this step and added the missing "click Submit" step too.